doc/compute_storage_schemes.rst

```diff
@@ -36,15 +36,15 @@ The script only makes use of Linux utilities that are available at provisioning
 	mkdir /sysroot/boot
 	mount /dev/sda1 /sysroot/boot
 
-Sometimes, it might be desired to format the compute disks using a different filesystem that is not included in initial ramdisk in which the partitioning script is called. In such cases it is required to update the ``dracut`` configuration in the osimage to include the desired filesystem support and utilities then pack it up.
+It may be necessary to format the compute disks using a different filesystem not included in the initial ramdisk called by the partitioning script. In such cases, update the ``dracut`` configuration in the osimage to include the desired filesystem support and utilities and then pack it up.
```

Comment:
should be the other way round, the partitioning script is run within the ramdisk